ministryofjustice/analytics-platform-control-panel
# App logs deep link

## :memo: Summary
This PR resolves https://github.com/ministryofjustice/analytical-platform/issues/4664

This PR adds a deep link from the manage app page to the logs in kibana

The changes in this PR are needed because we get a lot of support requests where checking the logs would have helped the user. However it is difficult for users to find relevant logs for their apps currently. This makes it easier, and make it easier to get users to check logs before asking for support.


Merging this PR will have the following side-effects:
- N/A

## :mag: What should the reviewer concentrate on?
- Link to logs

## :technologist: How should the reviewer test these changes?
- This is best tested once deployed to dev
- Log in (as superuser) and click the link to all apps
- Then click a "Manage app" button for an app
- Click the link to the logs for the relevant environment, it should take you to Kibana with the apps namespace, webapp container prefiltered

## :books: Documentation status
- [x] Documentation will be added in the future because ... User Guidance lives in a separate repo
